src/Enterspeed.Source.UmbracoCms.Base/Services/DataProperties/DefaultConverters/DefaultRichTextEditorPropertyValueConverter.cs

```diff
@@ -21,6 +35,22 @@ public bool IsConverter(IPublishedPropertyType propertyType)
         }
 
         public virtual IEnterspeedProperty Convert(IPublishedProperty property, string culture)
+        {
+            if (_httpContextAccessor.HttpContext == null)
+            {
+                using (_enterspeedHttpContextProvider.CreateFakeHttpContext())
```

Comment:
To avoid to much nesting, I think you can write it like this:

```
using (_enterspeedHttpContextProvider.CreateFakeHttpContext())
using (_umbracoContextAccessor.EnsureUmbracoContext())
{
    return ConvertWithContext(property, culture);
}
```